rosbag2_compression/src/rosbag2_compression/sequential_compression_writer.cpp

```diff
@@ -121,8 +121,10 @@ void SequentialCompressionWriter::open(
   if (storage_options.max_bagfile_size != 0 &&
     storage_options.max_bagfile_size < storage_->get_minimum_split_file_size())
   {
-    throw std::invalid_argument{
-            "Invalid bag splitting size given. Please provide a different value."};
+    std::stringstream error;
+    error << "Invalid bag splitting size given. Please provide a value greater than " <<
```

Comment:
Nit: do you want to add the size specified to the thrown message?

Reply:
I don't think that makes sense since it's already known by the user.
The goal of this PR is to inform the user of a suitable alternative to what they provided beforehand.